include/acutest.h

```diff
@@ -262,6 +262,12 @@ struct test__ {
     void (*func)(void);
 };
 
+enum {
+    TEST_FLAG_RUN = 1 << 0,
+    TEST_FLAG_SUCCESS = 1 << 1,
```

Comment:
The `enum` members make global identifiers. acutest.h uses suffix `__` to mitigate any risk of collision with sources implementing the tests, so please rename them accordingly.

Reply:
Done